sidecars/vale/Dockerfile

```diff
@@ -8,19 +8,20 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
+FROM jdkato/vale:v2.8.0 as vale
+
 FROM alpine:3.12.1
 
 ENV HOME=/home/theia
-ENV VALE_VERSION=2.4.0
+
+COPY --from=vale /bin/vale /usr/local/bin
```

Comment:
hello, why do we copy from an untrusted docker image the binary instead of the release as before ?
(because docker.io has limits so it might be an issue to use it)

Reply:
because the published binaries don't run on alpine. Using ldd, you can see that they are dynlinked to libc.